test/testUtils/mayaUtils.py

```diff
@@ -298,6 +298,16 @@ def mayaMajorMinorVersions():
     """
     return (mayaMajorVersion(), mayaMinorVersion())
 
+def ufeSupportFixLevel():
```

Comment:
Return the newly-implemented UFE fix level, to determine if the UFE pixelMove support is present in Maya or not.  fixLevel is implemented in Maya, in the ufeSupport.utils module.  If the fixLevel function does not exist, returns 0.